opentelemetry-instrumentation/src/opentelemetry/instrumentation/_semconv.py

```diff
@@ -470,4 +483,4 @@ def _set_status(
 def _get_schema_url(mode: _StabilityMode) -> str:
     if mode is _StabilityMode.DEFAULT:
         return "https://opentelemetry.io/schemas/1.11.0"
-    return SpanAttributes.SCHEMA_URL
+    return "https://opentelemetry.io/schemas/1.21.0"
```

Comment:
I've added 1.21.0 to the `opentelemetry.semconv.schemas.Schemas` enum so you can pick it from there instead of hardcoding

Reply:
Done.